mod/events.php

```diff
@@ -511,12 +516,34 @@ function events_content(App $a) {
 			'$preview' => t('Preview'),
 			'$acl' => $acl,
 			'$submit' => t('Submit'),
-			'$basic' => t("Basic"),
-			'$advanced' => t("Advanced"),
+			'$basic' => t('Basic'),
+			'$advanced' => t('Advanced'),
 			'$permissions' => t('Permissions'),
 
 		));
 
 		return $o;
 	}
+
+	// Remove an event from the calendar and its related items
+	if ($mode === 'drop' && $event_id) {
+		$del = 0;
+
+		$event_params = array('event_id' => ($event_id));
+		$ev = event_by_id(local_user(), $event_params);
+
+		// Delete only real events (no birthdays)
+		if (dbm::is_result($ev) && $ev[0]['type'] == 'event') {
+			$del = drop_item($ev[0]['itemid'], false);
+		}
+
+		if ($del == 0) {
+			notice( t('Failed to remove event' ) . EOL);
+		} else {
+			info( t('Event removed') . EOL);
```

Comment:
Standards: Please remove the space inside the parenthesis in the last two function calls.